### ISSUE TITLE
Check accessibility issues identified by SiteImprove

### ISSUE THREAD

**Issue by thepegisin** [cagov/covid19#5043]
Details: https://cadotgov.slack.com/archives/G01493HF6CU/p1628285898098800
